app/controllers/api/v1/albums_controller.rb

```diff
@@ -1,4 +1,7 @@
 class Api::V1::AlbumsController < ApplicationController
+  before_action :authenticate_user!, except: [:index, :show]
+  before_action :authorize_user, except: [:index, :show]
```

Comment:
it is better practice to be explicit in the actions we want, especially since this controller does not include all available actions. Instead, we should say `only: [:create, :destroy]`